helmfile/values/thanos/receiver.yaml.gotmpl

```diff
@@ -44,6 +48,10 @@ receive:
 
   mode: {{ .Values.thanos.receiver.mode }}
 
+  {{- if and (eq .Values.thanos.compactor.verticalCompaction true)  ( .Values.thanos.receiver.outOfOrderTimeWindow) }}
+  extraFlags:
+    - --tsdb.out-of-order.time-window={{ .Values.thanos.receiver.outOfOrderTimeWindow }}
+  {{- end }}
```

Comment:
if `outOfOrderTimeWindow` can only be used if `verticalCompaction` is enabled.
Can we have only `{{- if Values.thanos.compactor.verticalCompaction }}` as a condition and set a default value for `.Values.thanos.receiver.outOfOrderTimeWindow`?

Reply:
I mostly had it like that because if someone wanted to only have `verticalCompaction` enabled then they just had to not put in a value for `outOfOrderTimeWindow`. But do you think we should just always have `outOfOrderTimeWindow` enabled if `verticalCompaction` is enabled?

Reply:
based on this changelog entry I would say yes

> [#5839](https://github.com/thanos-io/thanos/pull/5839) Receive: Added parameter --tsdb.out-of-order.time-window to set time window for experimental out-of-order samples ingestion. Disabled by default (set to 0s). Please note if you enable this option and you use compactor, make sure you set the --enable-vertical-compaction flag, otherwise you might risk compactor halt.